.github/workflows/release.yaml

```diff
@@ -49,13 +49,6 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Install Rust Toolchain Components
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        target: ${{ matrix.target }}
```

Comment:
This line bugs me...  How can we communicate the target to the Rust Toolchain?

Reply:
Is `rustup` installed on the base image? If so, can we use `rustup` to install the correct toolchain?

Reply:
Yes, it is.  See the last line of #1873.  I know little about Rust tooling so you probably need to take over this pull request to get the `release` job finished.  It is cool to see that the `ci` job just works.